handbook/engineering/extensibility/index.md

```diff
@@ -0,0 +1,66 @@
+# Extensibility team
+
+The extensibility team owns our code host and third-party integrations (including our browser extension) and our [Sourcegraph extensions](https://docs.sourcegraph.com/extensions).
+
+## Contact
+
+- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
+- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+
+## Ownership
+
+This team is responsible for:
+
+1. Bringing the value of Sourcegraph to other developer tools
+1. Bringing the value of other developer tools into Sourcegraph
```

Comment:
Do these goals intentionally exclude any work on extensions that _don't_ integrate with some external developer tool, like your idea of the code tour extension @tjkandala? I personally think the goals should include/allow extensions like that if we want extensions to become a hit with customers in 2021

Reply:
[Same thought](https://github.com/sourcegraph/about/pull/2601#discussion_r580711709) :)